crates/nu-std/std/dt.nu

```diff
@@ -88,37 +88,33 @@ def borrow-second [from: record, current: record] {
     $current
 }
 
-def borrow-millisecond [from: record, current: record] {
-    mut current = $current
-    $current.microsecond = $current.microsecond + 1_000_000
-    $current.millisecond = $current.millisecond - 1
-    if $current.millisecond < 0 {
-        $current = (borrow-second $from $current)
-    }
-
-    $current
-}
-
-def borrow-microsecond [from: record, current: record] {
-    mut current = $current
-    $current.nanosecond = $current.nanosecond + 1_000_000_000
-    $current.microsecond = $current.microsecond - 1
-    if $current.microsecond < 0 {
-        $current = (borrow-millisecond $from $current)
-    }
-
-    $current
-}
-
-# Subtract two datetimes and return a record with the difference
-# Examples
-# print (datetime-diff 2023-05-07T04:08:45+12:00 2019-05-10T09:59:12+12:00)
-# print (datetime-diff (date now) 2019-05-10T09:59:12-07:00)
```

Comment:
Please leave this comment as this is what this script was written to do.

Reply:
@fdncred looks to me the new doc below might be better, right? :yum:

Reply:
No @amtoine, I disagree. The text below this doesn't have nushell's first commit date in the documentation. This is what I'm talking about `2019-05-10T09:59:12-07:00`. Again, that's why this was written and I don't want to lose track of that.

Reply:
ooh ok i see, yeah i guess the new doc can just use the Nushell first commit as the *earlier* date :+1:

Reply:
yes, and use `(date now)` as the later date

Reply:
Modified the example to use that date (exactly).  But first arg must be a constant, so we can display the result in help (and show the fields of the record at the same time).